src/figuresClasses.ts

```diff
@@ -1,19 +1,109 @@
 export interface Figure {
-
+  form: 'triangle' | 'circle' | 'rectangle',
+  color: 'red' | 'green' | 'blue',
 }
 
 export class Triangle {
+  constructor(
+    public color: Figure['color'],
+    public sideA: number,
+    public sideB: number,
+    public sideC: number,
+    public shape: Figure['form'],
+  ) {
+    if (sideA && sideB && sideC) {
+      let sides = [+sideA, +sideB, +sideC];
```

Comment:
```suggestion
      let sides = [sideA, sideB, sideC];
```
it is already number (you have TypeScript which says it)